charts/incubator/radicale/templates/_configmap.tpl

```diff
@@ -0,0 +1,160 @@
+    #timeout = 30
+    timeout = {{ .Values.radicale.server.timeout }}
+
+    # SSL flag, enable HTTPS protocol
+    #ssl = False
+
+    # SSL certificate path
+    #certificate = /etc/ssl/radicale.cert.pem
+
+    # SSL private key
+    #key = /etc/ssl/radicale.key.pem
+
+    # CA certificate for validating clients. This can be used to secure
+    # TCP traffic between Radicale and a reverse proxy
+    #certificate_authority =
+
+
+    [encoding]
+
+    # Encoding for responding requests
+    #request = utf-8
+    request = {{ .Values.radicale.encoding.request }}
+
+    # Encoding for storing local collections
+    #stock = utf-8
+    stock = {{ .Values.radicale.encoding.stock }}
+
+
+    [auth]
+
+    # Authentication method
+    # Value: none | htpasswd | remote_user | http_x_remote_user
+    #type = none
+
+    # Htpasswd filename
+    #htpasswd_filename = /etc/radicale/users
+
+    # Htpasswd encryption method
+    # Value: plain | bcrypt | md5
+    # bcrypt requires the installation of radicale[bcrypt].
+    #htpasswd_encryption = md5
+    htpasswd_encryption = bcrypt
+
+    # Incorrect authentication delay (seconds)
+    #delay = 1
+    delay = {{ .Values.radicale.auth.delay }}
+
+    # Message displayed in the client when a password is needed
+    #realm = Radicale - Password Required
+    realm = {{ .Values.radicale.auth.realm }}
+
+    [rights]
+
+    # Rights backend
+    # Value: none | authenticated | owner_only | owner_write | from_file
+    #type = owner_only
+    type = {{ .Values.radicale.rights.type }}
+
+    # File for rights management from_file
+    #file = /etc/radicale/rights
+
+
+    [storage]
+
+    # Storage backend
+    # Value: multifilesystem | multifilesystem_nolock
+    #type = multifilesystem
+    type = {{ .Values.radicale.storage.type }}
+
+    # Folder for storing local collections, created if not present
+    #filesystem_folder = /var/lib/radicale/collections
+    filesystem_folder = /data/collections
+
+    # Delete sync token that are older (seconds)
+    #max_sync_token_age = 2592000
+    {{- /*
+      Multiply by 1, so large integers aren't rendered in scientific notation
+      See: https://github.com/helm/helm/issues/1707#issuecomment-1167860346
+    */}}
+    max_sync_token_age = {{ mul .Values.radicale.storage.max_sync_token_age 1 }}
+
+    # Command that is run after changes to storage
+    # Example: ([ -d .git ] || git init) && git add -A && (git diff --cached --quiet || git commit -m "Changes by "%(user)s)
+    #hook =
+
+
+    [web]
+
+    # Web interface backend
+    # Value: none | internal
+    #type = internal
+    type = {{ .Values.radicale.web.type }}
+
+
+    [logging]
+
+    # Threshold for the logger
+    # Value: debug | info | warning | error | critical
+    #level = warning
+    level = {{ .Values.radicale.logging.level }}
+
+    # Don't include passwords in logs
+    #mask_passwords = True
+    mask_passwords = {{ .Values.radicale.logging.mask_passwords }}
```

Comment:
You can use the helm's ternary operator here `{{ ternary \"True\" "\False\" .Values.radicale.logging.mask_passwords }}`

And make it boolean on both values and questions. Not requirement, you decide :)

Reply:
Perfect, just what I was looking for! 👍 I was thinking of using the `title` function but was too lazy to test whether this works on `boolean`s.